packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.html

```diff
@@ -2,15 +2,15 @@
   <div class="slds-grid feedback__main-section">
     <span class="slds-var-p-right_medium slds-text-color_weak feedback__question"
       id="feedback__question">{question}</span>
-    <button aria-label={likeLabel} data-cy="feedback__like-button" aria-describedby="feedback__question" onclick={handleLike}
+    <button aria-label={likeLabel} data-cy="feedback__like-button" title={labels.thisAnswerWasHelpful} aria-describedby="feedback__question" onclick={handleLike}
```

Comment:
```suggestion
    <button aria-label={likeLabel} data-cy="feedback__like-button" title={likeLabel} aria-describedby="feedback__question" onclick={handleLike}
```

Reply:
See how the QuanticGeneratedAnswer component already sets the label correctly

Reply:
<img width="821" alt="image" src="https://github.com/coveo/ui-kit/assets/2488155/d2b2bb43-3f64-4f87-a3a0-4cd1903acb1a">